added a few links (#693)
graphprotocol/docs: website/pages/en/sunrise.mdx

title: Sunrise of Decentralized Data FAQ
---

> > Note: This document is continually updated to ensure the most accurate and helpful information is provided. New questions and answers are added on a regular basis. If you can’t find the information you’re looking for, or if you require immediate assistance, [reach out on Discord](https://discord.gg/vtvv7FP). If you are looking for billing information, then please refer to [billing](https://thegraph.com/docs/en/billing/).

Reviewer: Oh, also, that Discord invite link is old. The correct/evergreen one is `https://discord.gg/graphprotocol`. So:
```suggestion
> Note: This document is continually updated to ensure the most accurate and helpful information is provided. New questions and answers are added on a regular basis. If you can’t find the information you’re looking for, or if you require immediate assistance, [reach out on Discord](https://discord.gg/graphprotocol). If you are looking for billing information, then please refer to [billing](/billing/).
```

> > Note: This document is continually updated to ensure the most accurate and helpful information is provided. New questions and answers are added on a regular basis. If you can’t find the information you’re looking for, or if you require immediate assistance, [reach out on Discord](https://discord.gg/vtvv7FP). If you are looking for billing information, then please refer to [billing](https://thegraph.com/docs/en/billing/).

Author: Great catch!

### How can I ensure high quality of service and redundancy for subgraphs on The Graph Network?

> All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph and GRT](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).

Reviewer: "adding signal to your subgraph and GRT"? I don't get the phrasing. 🤔

Reviewer: Existed before where? I'm confused. Looks like you added "and GRT".

Author: Hi! Sorry, I confused it with another docs page phrasing.

Reviewer: No worries :) I see you removed it.

> All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph and GRT](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).

Author: ```suggestion
All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```
```suggestion
All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph and GRT](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```

### What happens if I don't upgrade my subgraph?

> Subgraphs will be queryable on the hosted service until June 12th. After this date, the hosted service homepage will still be accessible, however, query endpoints will no longer be available to query. Owners of hosted service subgraphs will still be able to upgrade their subgraphs to The Graph Network after June 12th, though earlier upgrades are entitled to [earn rewards](https://staging.thegraph.com/sunrise-upgrade-program/). Developers will also be able to claim [auto-upgraded subgraphs](https://thegraph.com/blog/unveiling-updated-sunrise-decentralized-data/#phase-2-sunbeam)

Reviewer: ```suggestion
Subgraphs will be queryable on the hosted service until June 12th. After this date, the hosted service homepage will still be accessible, however, query endpoints will no longer be available. Owners of hosted service subgraphs will still be able to upgrade their subgraphs to The Graph Network after June 12th, though earlier upgrades are entitled to [earn rewards](https://thegraph.com/sunrise-upgrade-program/). Developers will also be able to claim [auto-upgraded subgraphs](https://thegraph.com/blog/unveiling-updated-sunrise-decentralized-data/#phase-2-sunbeam)
```
Watch out for staging URLs. 😱
